app/controllers/people/new-position.js

```diff
@@ -0,0 +1,157 @@
+import Controller from '@ember/controller';
+import { action } from '@ember/object';
+import { inject as service } from '@ember/service';
+import { tracked } from '@glimmer/tracking';
+import { CLASSIFICATION_CODE } from 'frontend-contact-hub/models/administrative-unit-classification-code';
+
+const MANDATE = 'Mandaat';
+const MINISTER = 'Bedienaar';
+
+export default class PeopleNewPositionController extends Controller {
+  @service router;
+  @service store;
+
+  @tracked selectedOrganization;
+  @tracked selectedRole;
+  @tracked valid;
```

Comment:
Putting `@tracked valid = false;` would make us start in a disabled state because now it's enabled until we call `updateIsValid`